### ISSUE TITLE
Visible number of hidden tabs

### ISSUE THREAD

**Issue by llMaxll** [mozilla/multi-account-containers#2108]
Hi,
at the moment there is no way to distinguish a container with no open tabs from one with hidden tabs without navigating into them.
I would like to still see the number or at least a hint, that there are open but hidden tabs in the container overview.

**Comment by achernyakevich-sc** [comment]
the good company for mutli-account-containers is a Simple Tab Groups extension - https://addons.mozilla.org/en-US/firefox/addon/simple-tab-groups/

You could use it to have things grouped (including automated per container), as well you will not need to hid any tabs as these cold be moved to any invisible at the moment group.

**Comment by llMaxll** [comment]
You are right, that extension is awesome. Thank you so much for the recommendation. It makes switching the containers so much easier and also works better than Sticky Containers for keeping new tabs in the correct container.

I would like to still keep my feature request, since I think the hide/show container feature should work better for those who don't use Simple Tab Groups.
